2022q4/FreshPorts.md

```diff
@@ -0,0 +1,59 @@
+## Status Report Sample - This Will Become The Title ##
+
+Link:	 https://freshports.org/[FreshPorts]
+Link:	 https://news.freshports.org/[FreshPorts blog]
+
+Contact: Dan Langille <dan@FreeBSD.org>  
+
+FreshPorts, and its sister site, FreshSource, have reported
+upon FreeBSD commits for 20 years. They cover all commits,
+not just ports.
+
+FreshPorts tracks the commits and extracts data from the
+port Makefiles to create a database of information useful
+to both port developers and port users.
+
+For example, https://www.freshports.org/security/acme.sh/ shows
+the history of this port, back to its creation in May 2017.
+
+### converting the backend repository ###
+
+This topic deals with the FreshPorts code repository. The
+front end (website) was converted from subversion to git several
+years ago. The back end, which processes FreeBSD commits and
+updates the database, is still on subversion. I have
+wanted to convert these repositories to git for some time.
+
+I would like help with this please. I'll give you a copy of the 
+repositories and you give me back a git repo. It will be uploaded
```

Comment:
Clarification: do you want to merge several SVN repos into a single git repo?

Reply:
Multiple. Thanks.